app/resources/admin.html.template

```diff
@@ -138,559 +160,616 @@
-            {% ifequal view_config.min_query_word_length 1 %}
-              checked
-            {% endifequal %}
-        >
-        <label for="min_query_word_length_1">
-          Accept queries with only 1 character
-        </label>
-      </div>
-      <div class="option">
-        <input type="radio" name="min_query_word_length" value="2"
-            id="min_query_word_length_2"
-            {% ifnotequal view_config.min_query_word_length 1 %}
-              checked
-            {% endifnotequal %}
-        >
-        <label for="min_query_word_length_2">
-          Require query words to have at least 2 characters
-        </label>
-      </div>
-    </div>
-
-    <div class="config">
-      <div class="option">
-        <input type="radio" name="show_profile_entry" value="true"
-            id="show_profile_entry_true"
-            {{view_config.show_profile_entry|yesno:"checked,"}}>
-        <label for="show_profile_entry_true">
-          Show input fields for profile pages
-        </label>
-      </div>
-      <div class="option">
-        <input type="radio" name="show_profile_entry" value="false"
-            id="show_profile_entry_false"
-            {{view_config.show_profile_entry|yesno:",checked"}}>
-        <label for="show_profile_entry_false">
-          Hide input fields for profile pages
-        </label>
-      </div>
-    </div>
-
-    <div class="config">
-      <label for="profile_websites">External websites to show as the
-      default options for profile pages (JSON list of dictionaries sorted in the
-      order of display) <a href="javascript:insert_profile_websites_example()"
-      >click to insert an example</a>:</label>
-      <div class="response">
-        <textarea name="profile_websites" id="profile_websites" rows=3 cols=80
-          >{{view_config_json.profile_websites|default:"[]"}}</textarea>
-      </div>
-    </div>
-  </fieldset>
-
-  <fieldset>
-    <legend>Map options</legend>
-
-    <div class="config">
-      <label for="map_default_zoom">Default zoom level (integer):</label>
-      <div class="response">
-        <input name="map_default_zoom" id="map_default_zoom" size=24
-            value="{{view_config_json.map_default_zoom}}">
-      </div>
-    </div>
-
-    <div class="config">
-      <label for="map_default_center">
-        Default center [latitude north, longitude east]:
-      </label>
-      <div class="response">
-        <input name="map_default_center" id="map_default_center" size=24
-            value="{{view_config_json.map_default_center}}">
-      </div>
-    </div>
-
-    <div class="config">
-      <label for="map_default_center">
-        Map size in pixels [width, height]:
-      </label>
-      <div class="response">
-        <input name="map_size_pixels" id="map_size_pixels" size=24
-            value="{{view_config_json.map_size_pixels}}">
-      </div>
-    </div>
-
-  </fieldset>
+   function insert_profile_websites_example() {
```

Comment:
Do you know what made Github to think that there is a huge lines of code changed below? Did indentation change?